SourceX/DiabloUI/mainmenu.cpp

```diff
@@ -2,12 +2,12 @@
 
 int MainMenuResult;
 UI_Item MAINMENU_DIALOG[] = {
-	{ { 0, 0, 640, 480 }, UI_IMAGE, 0, 0, NULL, &ArtBackground },
-	{ { 64, 192, 510, 43 }, UI_LIST, UIS_HUGE | UIS_GOLD | UIS_CENTER, MAINMENU_SINGLE_PLAYER, "Single Player" },
-	{ { 64, 235, 510, 43 }, UI_LIST, UIS_HUGE | UIS_GOLD | UIS_CENTER, MAINMENU_MULTIPLAYER, "Multi Player" },
-	{ { 64, 277, 510, 43 }, UI_LIST, UIS_HUGE | UIS_GOLD | UIS_CENTER, MAINMENU_REPLAY_INTRO, "Replay Intro" },
-	{ { 64, 320, 510, 43 }, UI_LIST, UIS_HUGE | UIS_GOLD | UIS_CENTER, MAINMENU_SHOW_CREDITS, "Show Credits" },
-	{ { 64, 363, 510, 43 }, UI_LIST, UIS_HUGE | UIS_GOLD | UIS_CENTER, MAINMENU_EXIT_DIABLO, "Exit Diablo" },
+	{ { 0, 0, 640, 480 }, UI_IMAGE, UIS_NULL, 0, NULL, &ArtBackground },
+	{ { 64, 192, 510, 43 }, UI_LIST, (UiFlags)(UIS_HUGE | UIS_GOLD | UIS_CENTER), MAINMENU_SINGLE_PLAYER, "Single Player" },
+	{ { 64, 235, 510, 43 }, UI_LIST, (UiFlags)(UIS_HUGE | UIS_GOLD | UIS_CENTER), MAINMENU_MULTIPLAYER, "Multi Player" },
```

Comment:
Let's just change flags to an int instead.
https://github.com/diasurgical/devilutionX/blob/master/SourceX/DiabloUI/diabloui.h#L66